config/web/form.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Form\FormErrors;
+use Yiisoft\Form\FormErrorsInterface;
+
+return [
+    FormErrorsInterface::class => FormErrors::class,
```

Comment:
I don't think that `FormErrors` is a service. By having it in the container we make all forms sharing the same instance of `FormErrorsInterface`. That will likely result in a very weird behavior when having multiple forms.

Reply:
Each for should have its own errors.

Reply:
done.